p2p/CONFIG.md

```diff
@@ -0,0 +1,264 @@
+| -                      | Decodes the message and validate that it was sent on the right topic | A full validation pipeline, according to message type (according to topic). |
+
+**NOTE** Will be changed to be similar to ETH2.0 approach, 
+where validation pipelines will run on the topic validation level.
+expected to affect other configurations as well.
+
+More details on ETH2 message validators can be found 
+[here](https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/p2p-interface.md#global-topics)
+
+### Gossipsub: D
+
+Sets the optimal degree for a GossipSub topic mesh. D should be set somewhere between Dlo and Dhi.
+
+| Default Value (libp2p) | Default Value (SSV) | Default Value (ETH2) |
+| ---                    | ---                 | ---                  |
+| `6`                    | `8`                 | `8`                  |
+
+### Gossipsub: Dlo
+
+Sets the lower bound on the number of peers we keep in a GossipSub topic mesh.
+
+| Default Value (libp2p) | Default Value (SSV) | Default Value (ETH2) |
+| ---                    | ---                 | ---                  |
+| `5`                    | `6`                 | `6`                  |
+
+### Gossipsub: Dhi
+
+Sets the upper bound on the number of peers we keep in a GossipSub topic mesh.
+
+| Default Value (libp2p) | Default Value (SSV) | Default Value (ETH2) |
+| ---                    | ---                 | ---                  |
+| `12`                   | `12`                | `12`                 |
+
+### Gossipsub: HeartbeatInterval
+
+Controls the time between heartbeats, which are used across pubsub components to align on timing.
+
+| Default Value (libp2p) | Default Value (SSV) | Default Value (ETH2) |
+| ---                    | ---                 | ---                  |
+| `1s`                   | `700ms`             | `700ms`             |
+
+As SSV nodes work intensively with pubsub, we had to decrease the heartbeat interval 
+to reduce the latency created by a higher heartbeat.
+
+### Gossipsub: HistoryLength
+
+Controls the size of the message cache used for gossip (`IWANT` responses).
+
+| Default Value (libp2p) | Default Value (SSV) | Default Value (ETH2) |
+| ---                    | ---                 | ---                  |
+| `5`                    | `6`                 | `6`                  |
+
+The value was increased to reduce the number of `IWANT` messages in the network.
+
+### Gossipsub: HistoryGossip
+
+Controls how many cached message ids we will advertise in IHAVE gossip messages.
+
+| Default Value (libp2p) | Default Value (SSV) | Default Value (ETH2) |
+| ---                    | ---                 | ---                  |
+| `3`                    | `4`                 | `3`                  |
+
+The value was increased to ensure message delivery.
+
+### Gossipsub: MaxIHaveLength
+
+Sets the maximum number of messages to include in an IHAVE message.
+
+| Default Value (libp2p) | Default Value (SSV) | Default Value (ETH2) |
```

Comment:
why diff?

Reply:
Mentioned below: 
> Decreased value to avoid ihave floods, as our network produces a large number of messages.

> **NOTE** Will be increased once encoding changes to `SSZ` and compression as well (`snappy` or `s2`)